Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector.h

```diff
@@ -293,6 +293,16 @@ class VNL_EXPORT vnl_vector
   //: Iterator pointing to element beyond end of data
   const_iterator end() const { return data+num_elmts; }
 
+  //: Analog to std::vector::front().
```

Comment:
This should be analogous. Maybe update VLX with the spelling correction before doing sync?